src/test/java/reposense/util/TestUtil.java

```diff
@@ -199,7 +199,7 @@ public static boolean compareNumberFilesChanged(int expectedNumberFilesChanged,
      * Returns the file changed given a {@code rawFileChangedString}.
      */
     private static String getFileChanged(String rawFileChangedString) {
-        String fileChanged = rawFileChangedString.split("\\| ")[0].trim();
+        String fileChanged = rawFileChangedString.split("\t")[2].trim();
```

Comment:
Would it be suitable to reuse the constants `TAB_SPLITTER` and `STAT_FILE_PATH_INDEX` here?

Reply:
yep utilised them